packages/flutter/lib/src/material/chip.dart

```diff
@@ -97,6 +97,12 @@ abstract class ChipAttributes {
   /// {@macro flutter.widgets.Clip}
   Clip get clipBehavior;
 
+  /// {@macro flutter.widgets.Focus.focusNode}
+  FocusNode get focusNode;
```

Comment:
Here and elsewhere.

Is a focusNode always created? What happens if autofocus is true and focusNode is null? What if I don't want this widget to take the focus? The https://master-api.flutter.dev/flutter/widgets/Focus/focusNode.html doc doesn't mention this stuff.